### DISCUSSION TITLE
How can follow the cookiecutter update

### DISCUSSION THREAD

**Discussion by yswtrue** [cookiecutter/cookiecutter-django#4547] [General]
I have create a project, and how can I follow the update of cookiecutter, like the dependencies upgrade

**Comment by browniebroke** [comment]
See [this issue](https://github.com/cookiecutter/cookiecutter-django/issues/1047).

Basically: it's not supported by the cookiecutter CLI and this template doesn't do anything specific on that front. You might want to try out [cruft](https://cruft.github.io/cruft/), which is (AFAIU) a tool fully compatible with cookiecutter which supports updates.

**Reply by yswtrue** [reply]
Thank you

**Reply by dhoppe** [reply]
@browniebroke Is cruft still the right way to go or does a successor exist? As far as I know cruft has not seen any update for almost 2 years.
